src/encoded/schemas/workflow.json

```diff
@@ -222,17 +226,7 @@
                     "uniqueItems": true,
                     "items": {
                         "type": "string",
-                        "enum": [
-                            "c5.2xlarge",
-                            "c5.4xlarge",
-                            "m5a.2xlarge",
-                            "r4.4xlarge",
-                            "r5.4xlarge",
-                            "r5a.4xlarge",
-                            "r5b.4xlarge",
-                            "r5ad.4xlarge",
-                            "r6i.4xlarge"
-                        ]
+                        "pattern": "^[a-z][a-z0-9-]+[.][0-9]*[a-z-*]+$"
```

Comment:
I get the idea here but we may want a custom validator for this as if the intention is to validate for an expanded set of instances it will additionally validate instances that cannot often be used if at all (ie: GPU instances, Graviton etc).

Reply:
Agreed. Bioinformatics seemed to prefer the regex to the hard list, since adding a new instance would require an update and deployment, but there's more potential for using an unsupported instance or one with a typo with the regex. On CGAP, we never verified anything and did okay; will revisit as needed.